### ISSUE TITLE
Users: Save prefererred language

### ISSUE THREAD

**Issue by roschaefer** [roschaefer/rundfunk-mitbestimmen#183]
```gherkin
Feature: Save prefererred language
  As a registered user who prefers English
  I want to switch the language as soon as I log in
  In order to save me keystrokes and have a great user experience
```

*Note* 
If a user is authenticated, a language change should be saved in the backend. So the next time a user logs in, the application knows his or her preferred language and automagically change the language accordingly.

**Comment by mattlindsey** [comment]
@roschaefer I'd like to work on this issue. Can we discuss here or in an AgileV Scrum/Hangout?

**Comment by roschaefer** [comment]
Yes! That's a perfect issue for beginners.

Discussing in AgileV Scrum hangout has the following advantages:
* Face-to-face
* Communication goes fast

Discussing on Github has the advantages:
* Documenting the requirements
* Access the information anytime

I would suggest to get the best of both worlds in recording our AgileV scrum hangout and paste it here. Maybe with a short summary.

I have time tomorrow (Friday) and at the weekend.

**Comment by roschaefer** [comment]
closed via #277
